server/svix-server/src/core/cache/mod.rs

```diff
@@ -54,6 +54,25 @@ pub trait StringCacheValue: ToString + TryFrom<String> + Send + Sync {
 /// A macro that creates a [`CacheKey`] and ties it to any value that implements
 /// [`DeserializeOwned`] and [`Serialize`]
 macro_rules! kv_def {
+    ($key_id:ident, $val_struct:ident, $lit_prefix:literal) => {
```

Comment:
This is very redundant, can't you just make the other macro use this one? (even recursively using the same name, but also a different name if you can't use the same)